ios build fails on RN 0.72.6 with newArchEnabled

```
Undefined symbols for architecture arm64:
  "_RNVectorDrawableCls", referenced from:
      _RCTThirdPartyFabricComponentsProvider in libReact-RCTFabric.a(RCTThirdPartyFabricComponentsProvider.o)
ld: symbol(s) not found for architecture arm64
clang: error: linker command failed with exit code 1 (use -v to see invocation)

Undefined symbol: _RNVectorDrawableCls

Linker command failed with exit code 1 (use -v to see invocation)
```


the solution that worked for me:
renamed `RNVectorDrawableNativeComponent.ts` to `RNVectorDrawableNativeComponent.android.ts`

--- a/example/ios/Example.xcodeproj/project.pbxproj
+++ b/example/ios/Example.xcodeproj/project.pbxproj
@@ -572,13 +572,17 @@
 				);
 				MTL_ENABLE_DEBUG_INFO = YES;
 				ONLY_ACTIVE_ARCH = YES;
-				OTHER_CFLAGS = "$(inherited)";
+				OTHER_CFLAGS = (
+					"$(inherited)",
+					"-DRN_FABRIC_ENABLED",
+				);
 				OTHER_CPLUSPLUSFLAGS = (
 					"$(OTHER_CFLAGS)",
 					"-DFOLLY_NO_CONFIG",
 					"-DFOLLY_MOBILE=1",
 					"-DFOLLY_USE_LIBCPP=1",
 					"-DFOLLY_CFG_NO_COROUTINES=1",
+					"-DRN_FABRIC_ENABLED",
 				);
 				OTHER_LDFLAGS = (
 					"$(inherited)",
@@ -643,13 +647,17 @@
 					"\"$(inherited)\"",
 				);
 				MTL_ENABLE_DEBUG_INFO = NO;
-				OTHER_CFLAGS = "$(inherited)";
+				OTHER_CFLAGS = (
+					"$(inherited)",
+					"-DRN_FABRIC_ENABLED",
+				);
 				OTHER_CPLUSPLUSFLAGS = (
 					"$(OTHER_CFLAGS)",
 					"-DFOLLY_NO_CONFIG",
 					"-DFOLLY_MOBILE=1",
 					"-DFOLLY_USE_LIBCPP=1",
 					"-DFOLLY_CFG_NO_COROUTINES=1",
+					"-DRN_FABRIC_ENABLED",
 				);
 				OTHER_LDFLAGS = (
 					"$(inherited)",

--- a/example/ios/Example.xcworkspace/xcshareddata/IDEWorkspaceChecks.plist
+++ b/example/ios/Example.xcworkspace/xcshareddata/IDEWorkspaceChecks.plist
@@ -1,0 +1,8 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
+<plist version="1.0">
+<dict>
+	<key>IDEDidComputeMac32BitWarning</key>
+	<true/>
+</dict>
+</plist>

--- a/example/ios/Podfile
+++ b/example/ios/Podfile
@@ -5,6 +5,8 @@ require Pod::Executable.execute_command('node', ['-p',
     {paths: [process.argv[1]]},
   )', __dir__]).strip
 
+ENV['RCT_NEW_ARCH_ENABLED'] = '1'
+
 platform :ios, min_ios_version_supported
 prepare_react_native_project!
 

--- a/src/RNVectorDrawableNativeComponent.android.ts
+++ b/src/RNVectorDrawableNativeComponent.android.ts
@@ -1,10 +1,4 @@
-import {
-  ViewProps,
-  FlexStyle,
-  TransformsStyle,
-  ColorValue,
-  HostComponent,
-} from 'react-native';
+import { ViewProps, ColorValue, HostComponent } from 'react-native';
 
 // @ts-ignore
 import { WithDefault } from 'react-native/Libraries/Types/CodegenTypes';

